src/translations/nb.ts

```diff
@@ -164,6 +164,8 @@ const translations = {
   'stegvisning.sivilstand.radio_nei': 'Nei',
   'stegvisning.sivilstand.validation_error':
     'Du må svare på om du har samboer.',
+  'agepicker.validation_error.aar': 'Du må velge år og måned',
```

Comment:
Lurer på om brukere kan bli forvirret av "velge måned" (for det er jo antall måneder, ikke kalendermåned).
Forslag: 'Du må velge alder i år og måneder'

Reply:
Jeg spiller dette inn til Cecilie :)

Reply:
Svar fra Cecilie: "Snakket også om denne nå. Og vi er enig i at det er på en måte mer korrekt å be om "måneder", men det er på en annen side kun ett valg av måned man gjør, og skulle du velge 1 måned så blir det jo feil å bruke flertall der, så det blir på måte feil uansett, om vi formulerer det på den ene eller andre måten. Vi prøvde å oppdatere alle steder til måneder og synes det blir unødvendig tungt/langt. Hensikten med valideringen er at bruker skal forstå hva som skal til for å komme seg videre, og det tror vi at det gjør slik det er nå. Så vi ønsker å beholde det slik det er inntil vi ev. gjør noe funn som tilsier at det er forvirrende / et problem for bruker."